src/cpp/src/sequence_group.hpp

```diff
@@ -53,6 +53,8 @@ class Sequence {
     std::vector<std::vector<float>> m_generated_ids_embeds;
     SequenceGroupType m_type;
     size_t m_hidden_size;
+    std::vector<ov::Tensor> m_position_ids;
```

Comment:
How about using _list suffix to distinguish it is a vector?

Reply:
Agree, done.